### ISSUE TITLE
bin/gnu-pricing: line 37: -2: substring expression < 0

### ISSUE THREAD

**Issue by msdos** [diafygi/gnu-pricing#2]
When I try to do `cat --pricing` I get:

```
bin/gnu-pricing: line 37: -2: substring expression < 0
bin/gnu-pricing: line 47: -2: substring expression < 0
```

I'm using GNU bash, version 4.3.33(1)-release (x86_64-apple-darwin14.1.0).

**Comment by cschlosser** [comment]
Just remove the '-' in front of the 2 in line 37 and 47.
Should look like this:
ln37: `echo -e "$cmd\t$THISCOUNT\t\$${USECOST::2}.${USECOST:${#num}2}"`
ln47: `echo -e "Total\t$TOTALCOUNT\t\$${TOTALCOST::2}.${TOTALCOST:${#num}2}"`

Nvm it doesn't really.

This should be fixed in 333609fc0094ec7bcd638fa203360143b0dfdad5 
Needs some testing though.

**Comment by msdos** [comment]
I tested in zsh emulating bash (!!) and it works, as well as just bash. Thanks!
